### ISSUE TITLE
Mockups for alternative relabelings

### ISSUE THREAD

**Issue by aarppe** [UAlbertaALTLab/morphodict#1047]
1. Relabelings for lemma _wâpamêw_ with emojis

https://user-images.githubusercontent.com/15003407/150276848-d302d716-4cdc-4afa-909c-d9428b762e83.mov

2. Relabelings for wordform _niwâpamâw_ with emojis

3. Relabelings for lemma _wâpamêw_ without emojis

**Comment by aarppe** [comment]
@nienna73 Here are now elaborated mockups for relabing, with the three primary alternatives. The emoji for lexeme-specific info (here an **open book**) and the overall positioning of the information can well be discussed.

**Comment by nienna73** [comment]
This refactor is well underway. There's now one option that changes the relabelling globally. This can be done through the settings page or the options at the bottom of the page. Do we still want this one accessible to users?

![Screen Shot 2022-03-21 at 11 14 09 AM](https://user-images.githubusercontent.com/28357720/159327700-a5e24b0a-2099-429d-9ef1-d20ba018c19e.png)

**Comment by aarppe** [comment]
@nienna73 Yes, I can confirm we can still leave the selection of the relabeling scheme in conjunction with the paradigms, so it is explicitly visible there. Though we might decide later on to simplify things. What we probably would no longer need are the Community vs. Linguist Mode toggles at the bottom of the page (in part because they are sometimes difficult to reach), in part because we ought to add **nêhiyawêwin mode** there as well, for the sake completeness and consistency.
